INSTALL.md

```diff
@@ -327,20 +328,6 @@ files in the `lib` directory are likely to be corrupt or incomplete. Make
 sure that you used the proper archive and that extraction proceeded without
 errors.
 
-Especially try the command line editing and history facilities, because
-they are probably the most machine dependent feature of GAP. Enter a few
-commands and then make sure that `Ctrl-P` redisplays the last command, that
-`Ctrl-E` moves the cursor to the end of the line, that `Ctrl-B` moves the
-cursor back one character, and that `Ctrl-D` deletes single characters. So,
-after entering the above commands, typing
-
-    Ctrl-P  Ctrl-E  Ctrl-B  Ctrl-B  Ctrl-B  Ctrl-B  Ctrl-D  2  Return
```

Comment:
I think this whole paragraph does not belong into INSTALL.md at all. It reads like something for power users.